app/assets/javascripts/components/common/overview_stats_tabs.jsx

```diff
@@ -0,0 +1,64 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+
+import OverviewStatsTab from './OverviewStats/overview_stats_tab';
+import NamespaceOverviewStats from './OverviewStats/namespace_overview_stats';
+import WikidataOverviewStats from './wikidata_overview_stats';
+
+import { overviewStatsLabel } from '../../utils/wiki_utils';
+
+
+const OverviewStatsTabs = ({ statistics }) => {
+  const [currentTabId, setCurrentTabId] = useState(0);
+
+  const onTabChange = (e) => {
+    return setCurrentTabId(Number(e.currentTarget.id));
+  };
+
+  const statsList = [];
+  const tabsList = [];
+
+  let index = 0;
+  Object.keys(statistics).forEach((wiki_ns_key) => {
+    const statsTitle = overviewStatsLabel(wiki_ns_key);
+    const statsData = statistics[wiki_ns_key];
+
+    statsList.push({ statsTitle, statsData });
+    tabsList.push(
+      <OverviewStatsTab
+        key={index}
+        id={index}
+        onClick={onTabChange}
+        title={statsTitle}
+        active={currentTabId === index}
+      />
+    );
+    index += 1;
+  });
+
+  const title = statsList[currentTabId].statsTitle;
+  const data = statsList[currentTabId].statsData;
+  const content = (title === 'www.wikidata.org')
+    ? <WikidataOverviewStats statistics={data} isCourseOverview={true}/>
```

Comment:
It might make sense here to implement another simple component (eg, `OverviewStatsTab` (singular)), so that this tabs component has a simple job: render all the tabs and render the content for the current tab. The new component would be in charge of which component to render for the current tab content.

Reply:
Well, I have already created a component named `OverviewStatsTab` which renders a tab.
May be we can name it `OverviewStatsContent` or `OverviewStatsData` or anything like that. And, you are saying that the logic of identifying the content (whether it is wikidata or namespace stats) according to active tab, should be done in this component? Instead of having `statsList`, we should use this new component?

Reply:
You're right, `OverviewStatsTab` would be a bad name for it, and the component you have now called that is appropriately named. `OverviewStatsContent` sounds good.

And yes, I think having this component identify wikidata vs namespace stats should be done in that new component.

Reply:
@ragesoss, I have created the `OverviewStatsContent` component. Please check it along with `OverviewStatsTabs` to confirm if that's okay.

Reply:
yes, it looks good to me.